hray.txt

```diff
@@ -0,0 +1 @@
+hray - https://github.com/HughRaynor
```

Comment:
Have you registered for this workshop? We will not be merging any PRs which aren't part of our workshop